.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,6 +7,12 @@ assignees: ''
 
 ---
 
+<!--
```

Comment:
Not sure if it makes sense to add a troubleshooting guide link in the template for feature requests, people using this path shouldn't be having any issues to troubleshoot but asking for new requests. If that is not the case, the choose the wrong option to create the Issue